src/extraFiles/g_ether.conf

```diff
@@ -0,0 +1 @@
+options g_ether idVendor=0x04b3 idProduct=0x4010 iManufacturer="Raspberry Pi" bcdDevice=0x0100 iProduct="USB Ethernet Gadget" iSerialNumber=<serial>
```

Comment:
Are these VID and PID values significant, or are they arbitrarily chosen?

Reply:
They are significant - they have to match a known-good Windows-included USB ethernet driver. ISTR, these match to an IBM part.

Reply:
Btw. @tdewey-rpi I tried the ASUS ones but it did not work therefor I fall back to the IBM one.

Reply:
But I found some thing about a Linux Foundation USB ethernet gadget driver which should be in Windows. But I have to do a bit more research on this.